.circleci/config.yml

```diff
@@ -7,17 +7,11 @@ aliases:
   - &environment
     TZ: /usr/share/zoneinfo/America/Los_Angeles
 
-  - &restore_yarn_cache
```

Comment:
Renamed to clarify what this is actually restoring, which is the `node_modules` directory now.